From e6d10fa60430e61dadbc23eccb40daa0cc2b460a Mon Sep 17 00:00:00 2001
From: Don Olmstead <don.olmstead@sony.com>
Date: Wed, 29 Apr 2020 11:23:15 -0700
Subject: [PATCH 1/2] Remove postfix from archive name

CMake can't detect a shared build on Windows when there is a postfix version number.
---
 crypto/CMakeLists.txt | 2 +-
 ssl/CMakeLists.txt    | 2 +-
 tls/CMakeLists.txt    | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/crypto/CMakeLists.txt b/crypto/CMakeLists.txt
index 9b66d9f..4467ee7 100644
--- a/crypto/CMakeLists.txt
+++ b/crypto/CMakeLists.txt
@@ -1009,7 +1009,7 @@ if (WIN32)
 endif()
 set_target_properties(crypto PROPERTIES
 	OUTPUT_NAME crypto${CRYPTO_POSTFIX}
-	ARCHIVE_OUTPUT_NAME crypto${CRYPTO_POSTFIX})
+	ARCHIVE_OUTPUT_NAME crypto)
 set_target_properties(crypto PROPERTIES VERSION
 	${CRYPTO_VERSION} SOVERSION ${CRYPTO_MAJOR_VERSION})
 
diff --git a/ssl/CMakeLists.txt b/ssl/CMakeLists.txt
index cec2417..73763c9 100644
--- a/ssl/CMakeLists.txt
+++ b/ssl/CMakeLists.txt
@@ -86,7 +86,7 @@ if (WIN32)
 endif()
 set_target_properties(ssl PROPERTIES
 	OUTPUT_NAME ssl${SSL_POSTFIX}
-	ARCHIVE_OUTPUT_NAME ssl${SSL_POSTFIX})
+	ARCHIVE_OUTPUT_NAME ssl)
 set_target_properties(ssl PROPERTIES VERSION ${SSL_VERSION}
 	SOVERSION ${SSL_MAJOR_VERSION})
 
diff --git a/tls/CMakeLists.txt b/tls/CMakeLists.txt
index 7827a5e..a482af2 100644
--- a/tls/CMakeLists.txt
+++ b/tls/CMakeLists.txt
@@ -57,7 +57,7 @@ if (WIN32)
 endif()
 set_target_properties(tls PROPERTIES
 	OUTPUT_NAME tls${TLS_POSTFIX}
-	ARCHIVE_OUTPUT_NAME tls${TLS_POSTFIX})
+	ARCHIVE_OUTPUT_NAME tls)
 set_target_properties(tls PROPERTIES VERSION ${TLS_VERSION}
 	SOVERSION ${TLS_MAJOR_VERSION})
 
-- 
2.37.0.windows.1

